homeassistant/components/jewish_calendar/sensor.py

```diff
@@ -142,22 +143,18 @@
 )
 
 
-async def async_setup_platform(
+async def async_setup_entry(
     hass: HomeAssistant,
-    config: ConfigType,
+    config_entry: ConfigEntry,
     async_add_entities: AddEntitiesCallback,
-    discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Set up the Jewish calendar sensor platform."""
-    if discovery_info is None:
-        return
-
     sensors = [
-        JewishCalendarSensor(hass.data[DOMAIN], description)
+        JewishCalendarSensor(hass.data[DOMAIN][config_entry.entry_id], description)
```

Comment:
Please assign the hass.data[DOMAIN][config_entry.entry_id] to a variable before using it